terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/main/java/org/terasoluna/gfw/web/exception/ExceptionLoggingFilter.java

```diff
@@ -159,7 +159,7 @@ protected void logServletException(ServletException ex,
      */
     protected void logRuntimeException(RuntimeException ex,
             ServletRequest request, ServletResponse response) {
-        exceptionLogger.log(ex);
+        exceptionLogger.error(ex);
     }

```

Comment:
- [ ] All logging methods (`logIOException`, `logServletException`, `logRuntimeException`) do the same logic.
I think... you can unify logging methods, and use Catching Multiple Exception Types. What do you think?
```java
    @Override
    public void doFilter(ServletRequest servletRequest,
            ServletResponse servletResponse,
            FilterChain filterChain) throws IOException, ServletException {
        try {
            filterChain.doFilter(servletRequest, servletResponse);
        } catch (IOException | ServletException | RuntimeException e) {
            logException(e, servletRequest, servletResponse);
            throw e;
        }
    }

    // omitted...

    protected void logException(Exception ex, ServletRequest request, ServletResponse response) {
        exceptionLogger.error(ex);
    }
```